src/SFML/Audio/SoundFileReaderMp3.cpp

```diff
@@ -92,7 +92,7 @@ bool SoundFileReaderMp3::check(InputStream& stream)
 {
     std::uint8_t header[10];
 
-    if (static_cast<std::size_t>(stream.read(header, static_cast<std::int64_t>(sizeof(header)))) < sizeof(header))
+    if (stream.read(header, sizeof(header)) != sizeof(header))
```

Comment:
Is there a reason why this used to be `<`? Could `read` actually ever return more bytes than the header?

Reply:
I don't think the use of `<` here mattered. I think it was just an alternative way of saying "did we read the same number of bytes as the header?", in which case we ought to use `!=` like we do in many other places for this same thing.